<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateUpdatePage</class>
 <widget class="QWidget" name="CreateUpdatePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">.QPushButton{
	background-color: #67BA80;
	border: none
}

QWidget{
	font-size: 18px
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Тип компании</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="TypeInput">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <item>
      <property name="text">
       <string>ООО</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ОАО</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ЗАО</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ПАО</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Название кампании</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="NameInput"/>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Имя директора</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="BossNameInput"/>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Электронная очта</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="EmailInput"/>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Номер телефона</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="PhoneNumberInput">
     <property name="inputMask">
      <string>999 999 99 99</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Адрес</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="AddressInput"/>
   </item>
   <item>
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>ИНН</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="InnInput">
     <property name="inputMask">
      <string>9999999999</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Рейтинг</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="RateInput"/>
   </item>
   <item>
    <widget class="QPushButton" name="SaveButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="CancelButton">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Отменить</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
